src/Backend/Modules/Pages/Domain/ModuleExtra/ModuleExtraRepository.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Backend\Modules\Pages\Domain\ModuleExtra;
+
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Exception\ContentBlockNotFound;
+use Common\Locale;
+use Doctrine\ORM\EntityRepository;
+
+class ModuleExtraRepository extends EntityRepository
+{
+    public function add(ModuleExtra $moduleExtra): void
+    {
+        // We don't flush here, see http://disq.us/p/okjc6b
```

Comment:
that link doesn't work anymore actually. This is the blog post, but I'm not sure if we need to keep writing that there

https://matthiasnoback.nl/2014/05/inject-a-repository-instead-of-an-entity-manager/

Reply:
Removed in 4e0a27c2